cumulusci/tasks/github_commit.py

```diff
@@ -0,0 +1,188 @@
+import base64
+import hashlib
+import io
+import logging
+import os
+
+from cumulusci.core.exceptions import GithubException
+
+
+class CommitDir(object):
+    """Commit all changes in local_dir to branch/repo_dir"""
+
+    def __init__(self, repo, logger=None, author={}):
```

Comment:
Found some info on the concern here: http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/#mutable-default-arguments